(DOCS-5209) Add getting started with support (#17834)
DataDog/documentation: content/en/getting_started/support/_index.md

---

> title: Getting started with Datadog Support

Reviewer: ```suggestion
title: Getting Started with Datadog Support
```

kind: documentation
further_reading:
- link: "https://docs.datadoghq.com/agent/troubleshooting/"
  tag: "Documentation"
  text: "Agent Troubleshooting"
---

## Overview

Datadog provides two primary channels for customers seeking support:

>    - [Opening a support ticket][1].

Reviewer: I'm not sure why, but this link doesn't work for me in the staging preview (even though the link itself listed at the bottom of the file is correct).

Author: Thanks for flagging, resolved in [98fefe5].(https://github.com/DataDog/documentation/pull/17834/commits/98fefe52097bbf2d480e7a5c9d6af65712ee099f)

   - Chatting live with Datadog Technical Support Engineers through a text chat.

While both options focus on providing quick and effective solutions, one platform may be better suited depending on the timing, nature, and urgency of the request.

This guide provides best practices for reaching out to the Support team, and guidelines for determining the support channel that’s right for you.

## Prerequisites

For the most efficient resolution of an issue, be ready to provide any of the relevant information and resources listed below:

   - Organization name (if you have access to [more than one][2])
   - [Datadog Site][3]
   - Links to any pages demonstrating the issue in your Datadog account
   - Screenshots or screen recordings of the behavior in question
   - Steps that can be taken to replicate the issue
   - If the issue is related to the functioning of the Datadog Agent, an [Agent flare][4]
   - Links to any documentation pages used

## Chat or support ticket?

> Use the table below to determine whether to open a [support ticket][1] or reach out on chat. Chat is available any business day between the hours of 10:00 and 19:00 ET.

Reviewer: I'm not sure why, but this link doesn't work for me in the staging preview (even though the link itself at the bottom of the file is correct).

> Use the table below to determine whether to open a [support ticket][1] or reach out on chat. Chat is available any business day between the hours of 10:00 and 19:00 ET.

Reviewer: ```suggestion
Use the table below to determine whether to open a [support ticket][1] or reach out on chat. Chat is available any business day between the hours of 10:00 and 19:00 Eastern Time (ET).
```

| Support Ticket                  | Chat        |
| ------------------------------- | ----------- |
| Urgent issues                   | Product configuration support
| Call/Screenshare requests       | Setup questions
| Incidents and outages           | Documentation clarification
| Complex support troubleshooting across many configuration files, logs, or queries | Minor issues involving a specific configuration file, log, or query

If you're not sure which option is best, feel free to use either channel to connect with Datadog support. A support ticket is automatically created whenever a chat is closed, so the issue can continue to be investigated even if it's not resolved in chat.

## Reaching out on chat

> To get started, click on **Live Chat** on the left panel of your Datadog Account.

Reviewer: ```suggestion
To get started, click **Live Chat** on the left panel of your Datadog Account.
```

{{< img src="getting_started/chat/chat_button.png" alt="The chat button highlighted in Datadog" style="width:80%" >}}

When a new chat is opened, you are asked if you have a **technical** or **sales** question.
   - For technical questions, you are routed to the first available Technical Support Engineer. This is ideal for any questions you have about using or configuring Datadog.
   - For sales questions, you are routed to the first available member of our sales team, who can answer questions about billing and account management.

**Note**: For urgent issues, it’s best to reach out through a Support Ticket, and indicate why your request is urgent. This ensures that Support leadership can route your case to an appropriate expert immediately. 

### Best practices for technical questions

> Ensure to have as many of the relevant [prerequisites](#prerequisites) available as possible. The Datadog team member you connect with on chat does their best to collect information about the issue and resolve it. Not all issues can be resolved in a chat session. If a more detailed investigation is required, Datadog support focuses on collecting the information needed to conduct the investigation after the chat is over.

Reviewer: ```suggestion
Ensure that you have as many of the relevant [prerequisites](#prerequisites) available as possible. The Datadog team member you connect with on chat does their best to collect information about the issue and resolve it. Not all issues can be resolved in a chat session. If a more detailed investigation is required, Datadog support focuses on collecting the information needed to conduct the investigation after the chat is over.
```

---
title: Getting Started with Datadog Support
kind: documentation
further_reading:
- link: "https://docs.datadoghq.com/agent/troubleshooting/"
  tag: "Documentation"
  text: "Agent Troubleshooting"
---

## Overview

Datadog provides two primary channels for customers seeking support:
   - [Opening a support ticket][1].
   - Chatting live with Datadog Technical Support Engineers through a text chat.

While both options focus on providing quick and effective solutions, one platform may be better suited depending on the timing, nature, and urgency of the request.

> This guide provides best practices for reaching out to the Support team, and guidelines for determining the support channel that’s right for you.

Reviewer: ```suggestion
This guide provides best practices for reaching out to the Support team, and guidelines for determining the support channel that's right for you.
```

| Call/Screenshare requests       | Setup questions
| Incidents and outages           | Documentation clarification
| Complex support troubleshooting across many configuration files, logs, or queries | Minor issues involving a specific configuration file, log, or query

If you're not sure which option is best, feel free to use either channel to connect with Datadog support. A support ticket is automatically created whenever a chat is closed, so the issue can continue to be investigated even if it's not resolved in chat.

## Reaching out on chat

To get started, click **Live Chat** on the left panel of your Datadog Account.

{{< img src="getting_started/chat/chat_button.png" alt="The chat button highlighted in Datadog" style="width:80%" >}}

When a new chat is opened, you are asked if you have a **technical** or **sales** question.
   - For technical questions, you are routed to the first available Technical Support Engineer. This is ideal for any questions you have about using or configuring Datadog.
   - For sales questions, you are routed to the first available member of our sales team, who can answer questions about billing and account management.

> **Note**: For urgent issues, it’s best to reach out through a Support Ticket, and indicate why your request is urgent. This ensures that Support leadership can route your case to an appropriate expert immediately. 

Reviewer: ```suggestion
**Note**: For urgent issues, it's best to reach out through a Support Ticket, and indicate why your request is urgent. This ensures that Support leadership can route your case to an appropriate expert immediately. 
```
